invoices/update.go

```diff
@@ -214,11 +214,6 @@ func updateMpp(ctx *invoiceUpdateCtx, inv *Invoice) (*InvoiceUpdateDesc,
 	// Add amount of new htlc.
 	newSetTotal += ctx.amtPaid
 
-	// Make sure the communicated set total isn't overpaid.
-	if newSetTotal > ctx.mpp.TotalMsat() {
```

Comment:
If we delete this this check we need to change the check for "when this set is complete" from
```
setComplete := newSetTotal == ctx.mpp.TotalMsat() // line 241
```

to

```
setComplete := newSetTotal >= ctx.mpp.TotalMsat()
```

Reply:
Good catch!